playbook.yml

```diff
@@ -17,63 +17,84 @@
           - rsync
           - wget
 
-    - name: Install membres page code
-      git:
-        repo: https://github.com/epicerieledetour/ledetour-membres.git
-        dest: /root/ledetour-membres
-
-    - name: Update ledetour-membres website
-      synchronize:
-        src: /root/ledetour-membres/src/
-        dest: /srv/www/ledetour-membres
-        delete: yes
-        recursive: yes
-      delegate_to: "{{ inventory_hostname }}"
+    - name: Download charlesfleche keys
+      get_url: 
+        url: https://github.com/charlesfleche.keys
+        dest: .
+
+    - name: Download mauriandres keys
+      get_url: 
+        url: https://github.com/mauriandres.keys 
+        dest: .
+
+    - name: Download Wmbat keys
+      get_url:
+        url: https://github.com/Wmbat.keys 
+        dest: .
+
+    - name: Setup authorized public keys
+      shell: cat charlesfleche.keys mauriandres.keys Wmbat.keys > .ssh/authorized_keys
+
+    - name: Cleanup
+      shell: rm -f $(ls | grep ".keys")
+
+#    - name: Install membres page code
```

Comment:
You can remove those lines, no need to comment them, we can always go back in the Git history if required.